hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/auth/ProfileAWSCredentialsProvider.java

```diff
@@ -0,0 +1,56 @@
+package org.apache.hadoop.fs.s3a.auth;
```

Comment:
need the license...just copy from another class

Reply:
Fixed.